### PR TITLE
added schema, faqs and fixed image jump

### PR DESCRIPTION
**Pull request by anirudh-tantry**

### CODE REVIEW

File: components/template/TemplateCard.vue [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        class="img-fluid pointer template-preview__img"
+        :class="{ hidden: loading }"
+        :src="template.previewImageUrl"
+        :alt="template.name"
+        @load="loading = false"
+      />
+      <img
+        v-else
+        class="img-fluid pointer template-preview__img"
+        :class="{ hidden: loading }"
```
Comment (RAKSHITAMERIYA) [inline components/template/TemplateCard.vue:16]: The condition should be inverted for placeholder right?

File: components/template/Faq.vue [hunk 2]
```diff
@@ -0,0 +1,66 @@
+<template>
+  <section class="container py-2 py-lg-5">
+    <h2 class="text-center section__heading">Frequently Asked Questions</h2>
+    <div class="accordion accordion-flush my-5" id="accordionFaqs">
+      <div v-for="(faq, index) in faqs" :key="index" class="accordion-item">
+        <h2 class="accordion-header" :id="`heading${index}`">
```
Comment (RAKSHITAMERIYA) [inline components/template/Faq.vue:6]: Why :id="`heading${index}`"?

File: components/template/TemplateCard.vue (hunk 1 above)
Comment (anirudh-tantry) [reply components/template/TemplateCard.vue:16]: No, we are not showing placeholder image untill the actual image loads. we are showing a skeleton when image is loading

File: components/template/TemplateCard.vue [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        src="@/assets/images/templates/create_form.png"
+        alt="Template placeholder image"
+        @load="loading = false"
```
[the remark this answers is not in the document]
Comment (anirudh-tantry) [reply components/template/TemplateCard.vue:19]: as I mentioned above we are showing skeleton, so if the template doesn't have preview image then we will show placeholder image and hide the loading skeleton

File: components/template/Faq.vue (hunk 2 above)
Comment (anirudh-tantry) [reply components/template/Faq.vue:6]: we are using bootstrap accordion to differentiate which accordion to open. So index will differentiate the accordion

Comment (RAKSHITAMERIYA) [reply components/template/Faq.vue:6]: But I see below that on the button you are already providing data-bs-target, so is the above id required? Can you check once if it works even after removing it?

File: components/template/TemplateCard.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (RAKSHITAMERIYA) [reply components/template/TemplateCard.vue:19]: @anirudh-tantry in that case why not use a common image tag, as most of the properties are common, we can shift the src to a computed property named previewImageUrl whose value can be
`previewImageUrl() {
  return template.previewImageUrl || "@/assets/images/templates/create_form.png"
}`
alt could remain template.name for all of the image?

File: components/template/Faq.vue (hunk 2 above)
Comment (anirudh-tantry) [reply components/template/Faq.vue]: Yes its working

**Pull request merged**